opsdroid/connector/__init__.py

```diff
@@ -92,24 +165,49 @@ def configuration(self):
             bool: True for message successfully sent. False otherwise.
 
         """
-        _LOGGER.debug(_("%s connector can't react to messages"), self.name)
-        return False
+        warnings.warn(
+            "Connector.react is deprecated. Use "
+            "Connector.respond(events.Reaction(emoji)) instead.",
+            DeprecationWarning)
 
-    async def user_typing(self, trigger):
-        """Signals that opsdroid is typing.
+        return await message.respond(Reaction(emoji))
+
+    async def send(self, event):
+        """Send a message to the chat service.
 
         Args:
-            opsdroid (OpsDroid): An instance of the opsdroid core.
-            trigger: a bool that allows the event to be triggered on/off
+            event (Event): A message received by the connector.
+            target (string): The name of the room or other place to send the
+            event.
+
+        Returns:
+            bool: True for event successfully sent. False otherwise.
 
-        Triggers the "user is typing" event if the chat service that
-        opsdroid is connected to accepts it.
         """
+        if not isinstance(event, Event):
+            raise TypeError(
+                "The event argument to send must be an opsdroid Event object")
 
-    async def disconnect(self):
```

Comment:
Should we get rid of the disconnect method? (even though we don't have many connections implementing them)

Reply:
I haven't deleted it, just moved it up.

Reply:
I missed that sorry haha